### PR TITLE
BLOG: Multi-Cluster with Headless Services

### PR DESCRIPTION
**Pull request by CathPag**
@mateiidavid, could you have a look? Not sure how to handle hyperlinks and character limits.  Could you also add the images in the body? I'll find a thumbnail image in a bit

### CODE REVIEW

File: linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+description: 'We'll discuss two Linkerd features eagerly awaited by the community:  support for headless services and StatefulSets in multi-cluster environments. '
+keywords: []
+---
+
+In part two of our May Community Meeting recap blog, we'll focus on the
+presentation I delivered on two Linkerd features eagerly awaited by the
+community:  support for headless services and StatefulSets in multi-cluster
+environments. 
+
+I received great feedback and was asked to summarize it in this blog post.
```
Comment (cpretzer) [inline linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md:17]: How about: "All the great feedback from the community meeting inspired me to summarize the details in a blog post."

File: linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+[ADD IMAGE HERE]
+
+*(Fig 1.2: The structure of a replicated headless service)*
+
+Admittedly not the simplest solution, this approach does allow us to extend
+Linkerd's multi-cluster support to cover even more scenarios, especially for
+operators that run complex datastores and network topologies.
+
+This concludes the May community meeting recap. Don't wait for the next recap
+and [join our meetup live on Thursday, July 29 at 9 a,m, PT.](https://community.cncf.io/events/details/cncf-linkerd-community-presents-july-linkerd-online-community-meetup/)
```
Comment (cpretzer) [inline linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md:93]: `9 a,m` should be `9 AM`

**Review by cpretzer — CHANGES_REQUESTED** [review]
One small suggestion and a typo fix.

Do we have the images?

File: linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md (hunk 2 above)
Comment (mateiidavid) [reply linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md]: Fixed!

File: linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+title: Multi-Cluster Kubernetes with Headless Services
+subtitle: TMay Community Meeting Recap Part 2
+author: "matei"
+tags:
+  - Community
+  - Tutorials &amp; How-To's
+description: "We'll discuss two Linkerd features eagerly awaited by the community:  support for headless services and StatefulSets in multi-cluster environments."
+keywords: []
```
Comment (cpretzer) [inline linkerd.io/content/blog/multi-cluster-kubernetes-with-headless-services.md:9]: @CathPag this needs a date in order to be displayed

**Review by cpretzer — CHANGES_REQUESTED** [review]

**Review by cpretzer — APPROVED** [review]
LGTM

**Pull request merged**